Documentation/Configuration/Reference/TxSolrSearch.rst

```diff
@@ -841,7 +841,7 @@ The following example shows how to keep all options of all facets by keeping the
 plugin.tx_solr.search.faceting.keepAllFacetsOnSelection = 1
 plugin.tx_solr.search.faceting.countAllFacetsForSelection = 1
 plugin.tx_solr.search.faceting.minimumCount = 0
-```
+```````````````````````````````````````````````
```

Comment:
Not sure we want this change
Thats the code-block with three-backtick wrappers.

Reply:
I think we can/should skip this, the IDE did that automatically.

Reply:
Done